crates/uv-auth/src/middleware.rs

```diff
@@ -40,94 +40,55 @@ impl AuthMiddleware {
 impl Middleware for AuthMiddleware {
     async fn handle(
         &self,
-        mut req: Request,
-        _extensions: &mut Extensions,
+        mut request: Request,
+        extensions: &mut Extensions,
         next: Next<'_>,
     ) -> reqwest_middleware::Result<Response> {
-        let url = req.url().clone();
+        let url = request.url().clone();
 
         // If the request already has an authorization header, we don't need to do anything.
         // This gives in-URL credentials precedence over the netrc file.
-        if req.headers().contains_key(reqwest::header::AUTHORIZATION) {
-            debug!("Request already has an authorization header: {url}");
-            return next.run(req, _extensions).await;
+        if request
+            .headers()
+            .contains_key(reqwest::header::AUTHORIZATION)
+        {
+            debug!("Credentials are already present for {url}");
+            return next.run(request, extensions).await;
         }
 
-        // Try auth strategies in order of precedence:
-        if let Some(stored_auth) = GLOBAL_AUTH_STORE.get(&url) {
-            // If we've already seen this URL, we can use the stored credentials
-            if let Some(auth) = stored_auth {
-                debug!("Adding authentication to already-seen URL: {url}");
-                req.headers_mut().insert(
-                    reqwest::header::AUTHORIZATION,
-                    basic_auth(auth.username(), auth.password()),
-                );
-            } else {
-                debug!("No credentials found for already-seen URL: {url}");
```

Comment:
This logging is gone, though perhaps that's ok.

Reply:
Intentional, I found it too noisy. I'll be improving logging further in subsequent pull requests.